src/index.ts

```diff
@@ -391,7 +390,14 @@ const genHandler: ReqListener = (req, res) => { //? General handler
 	exStatic(req, res, nx)
 }
 
-const fileHandler: ReqListener = async (req, res) => { //? File handler
+const sendResponse = (res: Response, type: string, body: string | Buffer) => {
```

Comment:
You have already made custom methods like `res.typeLen()` and `res.log()`. Now create another one `res.send()` and move this code there instead of making it a separate function `sendResponse()` which anyways requires you to pass `res` in it.